From 915524e1ab7d2c5efb95985f30ee2f7c5ca69554 Mon Sep 17 00:00:00 2001
From: Rui Matos <tiagomatos@gmail.com>
Date: Wed, 02 May 2012 14:44:40 +0000
Subject: main: Close the recorder instead of pausing it

Pausing means that we will continue to use the same output file when
the keybinding is activated again. While useful to record a single
video in chunks, it doesn't seem to be how most users understand the
keybinding. Closing the recorder will close the file and create a new
one the next time the keybinding is pressed.

https://bugzilla.gnome.org/show_bug.cgi?id=675128
---
diff --git a/js/ui/main.js b/js/ui/main.js
index 3b80297..25ad8d8 100644
--- a/js/ui/main.js
+++ b/js/ui/main.js
@@ -110,7 +110,7 @@ function _initRecorder() {
         }
 
         if (recorder.is_recording()) {
-            recorder.pause();
+            recorder.close();
             Meta.enable_unredirect_for_screen(global.screen);
         } else {
             // read the parameters from GSettings always in case they have changed
--
cgit v0.9.0.2
